ui/packages/consul-ui/app/components/topology-metrics/source-type/index.scss

```diff
@@ -0,0 +1,4 @@
+.topology-metrics-source-type {
+  margin: 6px 0 6px 12px;
+  display: table;
```

Comment:
This jumps out as strange to me? I don't think we use this for any other badge type things?

Reply:
Yea, this is the first bad we have in the sandwiched between text. It needed a bit more spacing in the bottom and sides.